test-framework/arquillian/pom.xml

```
@@ -90,6 +90,27 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id >enforce</id>
+                        <configuration combine.self="append">
+                            <rules>
+                                <bannedDependencies>
+                                    <includes>
+                                        <exclude>junit:junit</exclude>
```

Shouldn't it be:
```suggestion
                                        <include>junit:junit</include>
```
?

---

It should. Weirdly though it still works, and you can actually use any tag name inside the <includes> as the parser must not verify the element name.